dev-docs/modules/floors.md

```diff
@@ -265,31 +265,204 @@ pbjs.setConfig({
 {% endhighlight %}
 
 
-### Floors Syntax
+## Floors Syntax
 
-The examples above covered several different scenarios. Here are all the options supported by the Floors module.
+The examples above covered several different scenarios where floors can be applied. Below we will cover the syntax and definition of the floors data schema. As of Prebid.js version 3.24, the Floors module supports a second data schema with the ability to add new schemas to future-proof the needs of additional design changes while keeping backwards compatibility.
+
+
+### Schema 1
+
+Schema 1 restricts floors providers or publishers to applying only one data group. To test more than one floor group, floor providers or publishers are required to reset the data set with new rules after each request bids.
 
 {: .table .table-bordered .table-striped }
 | Param | Type | Description | Default |
 |---+---+---+---+---|
 | enforcement | object | Controls the enforcement behavior within the Floors Module.| - |
-| enforcement.enforceJS | boolean | If set to true, the floor module will provide floors to bid adapters for bid request matched rules and suppress any bids not exceeding a matching floor. If set to false, the prebid floors module will still provide floors for bid adapters, there will be no floor enforcement.| true |
-| enforcement.enforcePBS | boolean | If set to true, the prebid js floors module will signal to PBS to pass floors to it’s bid adapters and enforce floors. If set to false, the pbjs should still pass matched bid request floor data to PBS, however no enforcement will take place. | false |
-| enforcement.floorDeals | boolean | Enforce floors for deal bid requests | false |
+| enforcement.enforceJS | boolean | If set to true, the floors module will provide floors to bid adapters for bid request matched rules and suppress any bids not exceeding a matching floor. If set to false, the prebid floors module will still provide floors for bid adapters, there will be no floor enforcement.| true |
+| enforcement.enforcePBS | boolean | If set to true, the Prebid.js floors module will signal to Prebid Server to pass floors to it’s bid adapters and enforce floors. If set to false, the pbjs should still pass matched bid request floor data to PBS, however no enforcement will take place. | false |
+| enforcement.floorDeals | boolean | Enforce floors for deal bid requests. | false |
 | enforcement.bidAdjustment | boolean | Adjust floors passed to Bid Aapters. If bid adjustment is passed to PBS and flag set is not set to false | true |
```

Comment:
The second sentence here (If bid adjustment...) is incomplete. Do you know what it's supposed to be?

Reply:
I really don't like the language i selected. Don't know what i was thinking. This document is so large I cannot recall my thought process at the time. I would like to reword this, with some assistance by you to wordsmith.

> If bidAdjustment is set to true, the Floors Module will adjust the applied floor per bidder by using bidAdjustment function if supplied. If set to false, floors will not be adjusted. Setting this flag to false may have unexpected consequence such as signaling a gross floor when expecting net or vice versa.

Reply:
Thanks for clarifying! Suggested rewording of the above:

"If true, the Floors Module will use the bidAdjustment function (if supplied) to adjust the floor per bidder. If false, floors will not be adjusted. Note: Setting this parameter to false may have unexpected results, such as signaling a gross floor when expecting net or vice versa."

What happens if they don't supply a bidAdjustment function?

Reply:
This is a per bidder setting the bidAdjustment. If the publisher does not supply a bidAdjustment function, then the Floors Module will do nothing to the bid. We can only adjust by what the bidAdjustment function provides.

Reply:
Okay, minor adjustment:

"If true, the Floors Module will use the bidAdjustment function to adjust the floor per bidder. If false (or no bidAdjustment function is provided), floors will not be adjusted. Note: Setting this parameter to false may have unexpected results, such as signaling a gross floor when expecting net or vice versa."

How does that sound?

Reply:
That looks good, thank you.